internal/pkg/database/queryer.go

```diff
@@ -0,0 +1,12 @@
+package database
+
+import (
+	"database/sql"
+)
+
+// Shared interface over both sqlx.Db and sqlx.Tx to enable
+// querying on both without considering the underlying type
+type Queryer interface {
```

Comment:
Doesn't sqlx have something like `orm.DB`, like go-pg?

Reply:
Nope, this is the "go way"